app/styles/print.scss

```diff
@@ -0,0 +1,101 @@
+@media print {
+  // Fixes the overflow during printing
+  #subsidyParent {
+    width: auto;
+    height: auto;
+    overflow: visible;
+  }
+  .au-c-app {
+    // overflow: visible;
+    height: auto;
+  }
+
+  .invisible-on-print {
+    visibility: hidden;
+  }
+
+  .display-on-print{
+    display: block !important;
+  }
+
+
+  // Hides the 'download bestand' button
+  .au-c-link[download] {
+    display: none;
+  }
+
+  // Show the title fully
+  .au-c-main-header__title--link {
+    overflow: visible;
+  }
+  // Hides the title href
+  .au-c-main-header__title--link:after {
+    content: none !important;
+  }
+
+  // show the logged in user fully
+  .au-c-dropdown__name[data-test-dropdown-title] {
```

Comment:
I wouldn't use those `data-test` attributes. Can't you simply add a new css class instead?

Reply:
I could but I'd have to add it in https://github.com/lblod/ember-submission-form-fields

Reply:
But this is custom code from the app, no? I don't think this is a form-field component?

Reply:
Sorry I was confusing it with something else, you're right I can just add a css class